### PR TITLE
Stats Date Picker - removed tooltip and made text always visible

### LINKED ISSUE #33831: Change cursor to pointer when hovering over tooltips
#### Steps to reproduce
1. Starting at URL: wordpress.com/stats/day/testingground728.blog
2. Screenshot: https://d.pr/i/CDxMS6
3. Hover over the small info icon
4. Set cursor to pointer. 

When I hover over a link I expect to see my cursor change, to help me know if I am on a link. In the case of a hover tooltip it's not required, since it shows up almost instantly. However, for the sake of consistency (all links use cursor: pointer), it may make sense to change this in CSS.

#### What I expected

Cursor to change to pointer in CSS

#### What happened instead

Cursor stays as default.

#### Browser / OS version

Chrome: Version 75.0.3770.80 (Official Build) (64-bit)
macOS Mojave Version 10.14.5

#### Screenshot / Video

https://d.pr/i/CDxMS6

### PR DESCRIPTION
**Pull request by MichaelArestad**
#### Changes proposed in this Pull Request

* Removes tooltip
* Migrates the tooltip text next to the "Last Updated" text

#### Testing instructions
* Go to `https://wordpress.com/stats/day/YOURSITE`

#### Before
image

#### After
image

Fixes #33831

### CODE REVIEW

File: client/my-sites/stats/stats-date-picker/index.jsx [hunk 1]
```diff
@@ -126,10 +124,12 @@ class StatsDatePicker extends Component {
 		const isToday = today.isSame( date, 'day' );
 		return (
 			<span>
-				{ translate( 'Last update: %(time)s', {
+				{ translate( '{{b}}Last update: %(time)s{{/b}} (Refreshes every 30 minutes)', {
```
Comment (scruffian) [inline client/my-sites/stats/stats-date-picker/index.jsx]: Would "Updates" be more user friendly and accurate than "refreshes"?

Comment (MichaelArestad) [reply client/my-sites/stats/stats-date-picker/index.jsx]: Yep. On it.

Comment (MichaelArestad) [reply client/my-sites/stats/stats-date-picker/index.jsx]: Changed.

**Comment by sixhours** [comment]
Works a charm! 👍

**Pull request merged**